protocol/activity_center_persistence.go

```diff
@@ -21,6 +22,25 @@ func (db sqlitePersistence) DeleteActivityCenterNotificationByID(id []byte, upda
 	return err
 }
 
+func findNextLastMessage(notifications []*ActivityCenterNotification, messageID string) *common.Message {
```

Comment:
in mobile it's used to contact request state 

https://github.com/status-im/status-mobile/blob/ba16e1adcc0fe7e26fa636e4851ee3c90de86dc7/src/status_im/contexts/shell/activity_center/notification/contact_requests/view.cljs#L170

I'm not sure if this can be read from chat-id then chat last-message @ilmotta any thoughts?

Reply:
I don't have the full picture, but perhaps that code I wrote on mobile a long time ago using the last message is not needed anymore. If I remember correctly, the last message was used for chat notifications even before we implemented the new Activity Center, but that's not used anymore. Maybe we can just get rid of last message? I can't be sure.